SwiftStaticLibrarySample/ViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ViewController.swift
+//  SwiftStaticLibrarySample
+//
+//  Created by Chad Landis on 8/7/17.
+//  Copyright © 2017 Chad Landis. All rights reserved.
+//
+
+import UIKit
+
+// Import Dynamic framework, this is totally normal.  We can access both Swift and Objective-C code
+// that is exported from the module (all Objective-C code in the module header, and all public Swift)
+import SwiftDynamicFramework
+
+// Import Static library, this is totally normal.  We can access both Swift and Objective-C code
+// that is exported from the module (all Objective-C code in the module header, and all public Swift)
+// However, this import statement generates the following warning:
+// "Implicit import of bridging header 'SwiftStaticLibrary.h' via module 'SwiftStaticLibrary' is deprecated and will be removed in a later version of Swift"`"
```

Comment:
This warning is still showing up in Xcode 9 beta 6.  Is this expected?  See the comment for more details, but I don't get a similar warning in the [test target](https://github.com/calandis/SwiftStaticLibrarySample/pull/1/files#diff-f96a6905ef790a8338948772bd6b4e82R10).